### PR TITLE
Set up CORS for the flask server

### PR DESCRIPTION
**Pull request by CKrawczyk**
This adds CORS for any zooniverse.org domain on POST methods.

### CODE REVIEW

File: panoptes_aggregation/routes.py [hunk 1]
```diff
@@ -69,6 +70,15 @@ def make_application():
                         static_url_path='',
                         static_folder='../docs/build/html')
     application.json_encoder = MyEncoder
+    CORS(
+        application,
+        origins=[
+            r'^https?:\/\/([a-z0-9-.]+zooniverse.org)'
```
Comment (wgranger) [inline panoptes_aggregation/routes.py]: Is it possible to add localhost here for development purposes?

Comment (CKrawczyk) [reply panoptes_aggregation/routes.py]: Sure, I assume a regex similar to this one form Panoptes will work https://github.com/zooniverse/Panoptes/blob/master/config/cors_config.yml.hudson#L7

Comment (camallen) [reply panoptes_aggregation/routes.py]: I'd prefer we use `local.zooniverse.org` for local dev.

File: panoptes_aggregation/routes.py [hunk 2]
```diff
@@ -69,6 +70,16 @@ def make_application():
                         static_url_path='',
                         static_folder='../docs/build/html')
     application.json_encoder = MyEncoder
+    CORS(
+        application,
+        origins=[
+            r'^https?:\/\/([a-z0-9-.]+zooniverse.org)',
+            r'^https?:\/\/(127\.0\.0\.1|localhost|[a-z0-9-]+\.local)(:\d+)?$'
```
Comment (camallen) [inline panoptes_aggregation/routes.py]: i'd prefer we use `local.zooniverse.org` for local dev on production system. If this system can't leak any private data then it's probably ok for localhost as well.

File: panoptes_aggregation/routes.py [hunk 3]
```diff
@@ -69,6 +70,16 @@ def make_application():
...
+        ],
+        methods=[
+            'POST'
```
Comment (camallen) [inline panoptes_aggregation/routes.py]: Will you need OPTIONS here as well? It may not be but all preflight CORS requests are OPTIONS, I'm not sure

Comment (CKrawczyk) [reply panoptes_aggregation/routes.py]: I will add it in to be safe.

File: panoptes_aggregation/routes.py (hunk 2 above)
Comment (CKrawczyk) [reply panoptes_aggregation/routes.py]: @wgranger do you have `local.zooniverse.org` set up on you system (e.g. https://stackoverflow.com/c/zooniverse/questions/109) if so I will adjust the first regex to work when a port is put on the end.

**Pull request merged**